Sources/KeyboardSupport/KeyboardNavigator.swift

```diff
@@ -21,8 +21,15 @@ public protocol KeyboardNavigatorDelegate: class {
 @available(*, deprecated: 2.0, renamed: "KeyboardNavigator")
 public typealias KeyboardManager = KeyboardNavigator
 
+public typealias UITextInputView = UIView & UITextInput
+
+public protocol KeyboardNavigating {
```

Comment:
What about making KeyboardNavigating a struct with properties for keyboardToolbar and returnKeyNavigationEnabled? This would reduce code in both KeyboardNavigator and KeyboardAutoNavigator.

Reply:
I'm not opposed to making it a parent class instead of a protocol.

Reply:
I created a "KeyboardNavigatorBase" class that will act as the parent of the KeyboardNavigators. I went with this name in an effort to not break current users of the KeyboardNavigator class. 

https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/b4e7789de586889bc7ff822b416b890c4e9408bf